test/modules/ModuleLib/ReporterClients/TestResultCoordinatorReporterClient.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.ModuleUtil.ReporterClients
+{
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil.TestResults;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    public sealed class TestResultCoordinatorReporterClient : ReporterClientBase
```

Comment:
since it is used for reporting to testanalyzer and trc; please rename it.
```suggestion
    public sealed class TestResultReporterClient : ReporterClientBase
```